curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -342,16 +342,20 @@ MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest& request) {
         needUpdate = true;
     }
 
+    bool fileNeedDeallocate =
```

Comment:
do we have trash for volume file?

Reply:
Trash was originally designed to delete s3info or extent. the deletion of S3info is relatively simple, but the deletion of volumeExtent need consider more，so it is encapsulate in InodeVolumeSpaceDeallocate.